dnf5.spec

```diff
@@ -184,6 +184,33 @@ It supports RPM packages, modulemd modules, and comps groups & environments.
 %license COPYING.md
 %license gpl-2.0.txt
 %{_mandir}/man8/dnf5.8.*
+%{_mandir}/man8/dnf5-advisory.8.*
+%{_mandir}/man8/dnf5-clean.8.*
+%{_mandir}/man8/dnf5-distro-sync.8.*
+%{_mandir}/man8/dnf5-downgrade.8.*
+%{_mandir}/man8/dnf5-download.8.*
+%{_mandir}/man8/dnf5-environment.8.*
+%{_mandir}/man8/dnf5-group.8.*
+# TODO(jkolarik): history is not ready yet
+# %{_mandir}/man8/dnf5-history.8.*
```

Comment:
These lines make rpm complain about macro expansion in comment. But we can live with it for a while, I guess.